packages/shared/src/components/Feed.tsx

```diff
@@ -347,6 +358,22 @@ export default function Feed<T>({
       },
     });
 
+    if (!!user && !post.bookmarked && !seenBookmarkPromotion) {
+      const bookmarkLoops = await growthbook.getFeatureValue(
```

Comment:
Not super blocking, but is there anyway we can move this as a function inside the GB provider or lib?
It would make it easier to not have to import multiple dependencies for each case.

Reply:
@sshanzel If we can easily avoid calling the clear function (maybe something like if it's a app defined CTA, so part of the object you made) don't fire it?

Reply:
Yeah, I've thought of that too, also that we already have the default value at the same time in the featureManagement. Would at least avoid mistakes of putting a different default value.

> @sshanzel If we can easily avoid calling the clear function (maybe something like if it's a app defined CTA, so part of the object you made) don't fire it?

Yeah we can do that, should be a simple find and we should be good.